tests/test_devices.py

```diff
@@ -0,0 +1,19 @@
+from parla import Parla
+from parla.cython import device 
+
+CUDADeviceBinder = device.CUDADeviceBinder
```

Comment:
Is this test updated? Is `CUDADeviceBinder` now `PyCUDADevice`?

Reply:
This test is deprecated as the device manager is now created by the scheduler. The device manager prints registered devices while the scheduler completes its registration.